Makefile

```diff
@@ -4,7 +4,7 @@ TAGS ?= 81 81-fpm 81-unit 82 82-fpm 82-unit
 COMPOSER_HASH ?= 55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae
 DRUSH_VERSION ?= 8.4.12
 DOCKER_BUILDKIT ?= 1
-PLATFORM ?= linux/amd64
+PLATFORM ?= linux/amd64,linux/arm64
```

Comment:
Any reason to provide 2 platforms when it possible to build only one?

Reply:
The reason is to specify/document in code all supported platforms.